pkgs/phps.nix

```diff
@@ -45,6 +45,15 @@ let
           else
             super.extensions.mysqlnd;
 
+        oci8 =
+          if prev.lib.versionOlder super.php.version "7.0" then
+            super.extensions.oci8.overrideAttrs (attrs: {
```

Comment:
The issue with PHP 5.6 is actually that you are overriding the arguments to `mkDerivation`, so the `src` will not actually be updated. You need to replace `overrideAttrs` with `override`.

Reply:
I just noticed this as you commented :smile:

Currently building against all `php` versions after I updated `flakes` :+1: